pkg/azurefile/controllerserver.go

```diff
@@ -117,16 +121,13 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	enableHTTPSTrafficOnly := true
 	shareProtocol := storage.SMB
 	var vnetResourceIDs []string
-	if fsType == nfs || protocol == nfs {
-		protocol = nfs
+	if protocol == nfs {
```

Comment:
just for my own learning, when should nfs be used vs cifs?

Reply:
@andyzhangx I also want to learn :)

Reply:
Azure file provides both SMB(by default) and NFS protocols

Reply:
Yeah, but is there a recommendation on when to use NFS over cifs? Normally how should user decides which one to use?

Reply:
currently cifs is the default protocol